azure-pipelines/azure-templates/build-template.yml

```diff
@@ -0,0 +1,51 @@
+
+parameters:
+  - name: 'displayName'
+    type: string
+  - name: 'mavenPomFile'
+    type: string
+  - name: 'options'
+    default: ''
+    type: string
+  - name: 'targetPath'
+    type: string
+  - name: 'artifact'
+    type: string
+
+
+steps:
+- task: SonarCloudPrepare@1
```

Comment:
be consistent about indentation of yaml sequences. e.g. inconsistent between here and the parameters just above. same for all yaml files here.

Reply:
Deprecated. Removed.